contracts/javascore/xcall/src/main/java/foundation/icon/btp/xcall/CallServiceImpl.java

```diff
@@ -114,36 +101,38 @@ private void cleanupCallRequest(BigInteger sn) {
     public BigInteger sendCallMessage(String _to, byte[] _data, @Optional byte[] _rollback) {
         BigInteger sn = getNextSn();
         Address caller = Context.getCaller();
-        Context.require(caller.isContract() || _rollback == null, "RollbackNotPossible");
+        Context.require(caller.isContract(), "Only contract is allowed to send call message ");
```

Comment:
This change should be reverted. Caller only need to be contracts if rollback is passed
https://github.com/icon-project/IIPs/issues/52#issuecomment-1384866064

Reply:
done